.cloudbuild/run_deploy.sh

```diff
@@ -0,0 +1,18 @@
+# /bin/bash
+
+gcloud run deploy ${_SERVICE} \
+    --platform managed --region ${_REGION} \
+    --image gcr.io/$PROJECT_ID/${_SERVICE}  \
+    --allow-unauthenticated  \
+    --add-cloudsql-instances ${_DATABASE_INSTANCE} \
+    --service-account cloudrun-berglas-python@${PROJECT_ID}.iam.gserviceaccount.com \
+    --update-env-vars DATABASE_URL=berglas://${_BERGLAS_BUCKET}/database_url,SECRET_KEY=berglas://${_BERGLAS_BUCKET}/secret_key,GS_BUCKET_NAME=berglas://${_BERGLAS_BUCKET}/media_bucket
+
+# This is annoying to have to branch out just because of this stip.  I'd love to do this inline. 
+SERVICE_URL=$(gcloud run services list --platform managed --format="value(status.url)" --filter="metadata.name=${_SERVICE}")
+
+echo "Setting ${_SERVICE} CURRENT_HOST to <$SERVICE_URL>."
+
+gcloud run services update ${_SERVICE}  \
+    --platform managed --region ${_REGION}\
+    --update-env-vars CURRENT_HOST=${SERVICE_URL}
```

Comment:
Potentially could load this from the Cloud Run Admin API on startup. Looks like an odd bit of configuration.

Reply:
Do you know of a way to get the hash in a service URL without getting it from an existing service? That's the concept here: I don't know the service URL until the service is deployed, but if I knew the project hash, I could construct it (given I know the region and service name). This seemed like a simpler workaround, to update the value after it's known to exist (this script is designed to run on first create and subsequent updates.)

Reply:
Unfortunately there is not, but it is unique by project + region. @ptone has experimented with using the hash as a parameter.